src/main/java/babric/BabricRepositoryHandler.java

```diff
@@ -1,17 +0,0 @@
-package babric;
-
-import org.gradle.api.artifacts.dsl.RepositoryHandler;
-
-public class BabricRepositoryHandler {
-    protected static void declareRepositories(RepositoryHandler repositories) {
-        repositories.maven(repo -> {
```

Comment:
Can you add back Babric repo too in the new repo declaration class?

Reply:
It's used as you don't have to declare Babric maven in your own project this way. Because one developing mods for Babric is very likely to make use of it. Babric loader and mappings being an example of it.